controllers/chaosimpl/jvmchaos/impl.go

```diff
@@ -139,8 +141,9 @@ func (impl *Impl) Recover(ctx context.Context, index int, records []*v1alpha1.Re
 		EnterNS:     true,
 	})
 	if err != nil {
+		// Unable to uninstall the JVMRules, most likely because the jvm process has been restarted
 		impl.Log.Error(err, "uninstall jvm rules")
-		return v1alpha1.Injected, err
+		return v1alpha1.NotInjected, nil
```

Comment:
ditto

Reply:
Here I can only see the exit status of 1.

Reply:
OK, I will update the error next pr. How about reverting this update?

Reply:
I file another pr https://github.com/chaos-mesh/chaos-mesh/pull/3004, after this pr merged, the error will contain the real failed reason. How about update this pr after my pr is merged?

Reply:
Sure,  will do

Reply:
I have added the check for the real failed reason (Connection refused)

Reply:
Great！